wgpu-core/src/device/mod.rs

```diff
@@ -3484,9 +3464,11 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                     pipeline::ShaderModuleSource::Wgsl(ref code) => {
                         trace.make_binary("wgsl", code.as_bytes())
                     }
-                    pipeline::ShaderModuleSource::Naga(_) => {
-                        // we don't want to enable Naga serialization just for this alone
-                        trace.make_binary("ron", &[])
+                    pipeline::ShaderModuleSource::Naga(ref module) => {
```

Comment:
Missed a SpirV above this branch

Reply:
There isn't any SpirV in there any more. Could you clarify, please?

Reply:
Apparently not, nvm then :)